atomix/cluster/src/test/java/io/atomix/raft/zeebe/ZeebeLogAppenderTest.java

```diff
@@ -103,24 +103,6 @@ public void shouldNotifyOnError() {
     assertEquals(0L, appenderListener.getCommitted().size());
   }
 
-  @Test(timeout = 5000)
-  public void shouldNotAppendInconsistent() {
```

Comment:
Because it was replaced by better tests. The check depends on the listener so now we have tests to test this from the listener perspective (LogStorageAppenderTest) and also from the LeadeRole perspective (LeaderRoleTest). I think the tests are more thorough as well

Reply:
ok thanks